app/src/components/pages/ScheduleGenerate.tsx

```diff
@@ -39,24 +45,45 @@ function ScheduleGenerate() {
     (s) => JSON.parse(s)
   );
 
-  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    const output = {
-      term: term,
-      classes: classes
-    };
-    console.log(output);
-    e.preventDefault();
-    navigate('/schedule');
-    //TODO: submit values somewhere
-  };
+  const [submitHandler, { data, loading, error }] = useMutation(GENERATE_SCHEDULE);
+
+  if (data) {
+    console.log(data);
+    console.log(classes);
+    return (
+      <Dialog
+        open={true}
+        keepMounted
+        onClose={() => window.location.href = '/schedule/generate'}
+        aria-describedby="alert-dialog-slide-description"
+      >
+        <DialogTitle>{"Submission Successful"}</DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-slide-description">
+            Your generation request was submitted successfully. 
+            When the scheduling algorithm completes, you'll be able to view the schedule on the management page.
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions>
+          <Button onClick={() => window.location.href = '/schedule/generate'}>Ok</Button>

```

Comment:
Using `.href` because I'm too lazy to figure out proper state management and use `navigate()`. Not good form but works and refreshes the component completely

Reply:
Bro is code reviewing his own code